spec.md

```diff
@@ -84,17 +84,18 @@ added to enable support for new features.
 These fields MUST be OPTIONAL and SHOULD be ignored by clients and servers
 that do not understand them.
 
-### Changes Since v2.10
```

Comment:
We should add a link to the release notes here with text like 'For changes in older versions, see the release notes'